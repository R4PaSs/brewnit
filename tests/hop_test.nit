import dummy_db

with ctx = new DBContext do
	var res = ctx.hop_worker.fetch_multiple(ctx, "* FROM hops")
	for i in res do print i
end
