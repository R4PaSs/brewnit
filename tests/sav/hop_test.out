Hop: Hallertau Mittlefruer
Hop: Fuggles
Hop: Simcoe
